index.html

```diff
@@ -268,7 +323,45 @@ <h2 class="javaS">CSS</h2>
           </div>
         </div>
       </div>
-    
-  </section>
+      <footer>
+        <div class="contact-me-img">
+          <img
+            class="contactme-img"
+            src="Microverse-Student-Project-1/Contact.png"
+            alt=""
+          />
+          <img
+            class="contact-shapes"
+            src="Microverse-Student-Project-1/Contact2.png"
+            alt=""
+          />
+        </div>
+        <div class="contact-text">
+          <h1 class="Contactme">Contact Me</h1>
+        </div>
+        <div class="contactmepara">
+          <p class="contactme-p">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
 
-</body>
\ No newline at end of file
+        <form action="https://formspree.io/f/contact-information" method="post" class="contact-information" id="contact-information" >
+          <input class="name-input" type="text" name="username" placeholder="alzubair" maxlength="30" required>
```

Comment:
[OPTIONAL]
This input field isn't properly closed. Kindly close the input field 👍🏻

```
<input class="name-input" type="text" name="username" placeholder="alzubair" maxlength="30" required />
```